source/patterns/@aws-solutions-constructs/core/lib/mediastore-defaults.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ *  Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import * as cdk from '@aws-cdk/core';
+import * as mediastore from '@aws-cdk/aws-mediastore';
+
+export function MediaStoreContainerProps(): mediastore.CfnContainerProps {
+  return {
+    containerName: cdk.Aws.STACK_NAME,
+    accessLoggingEnabled: true,
+    corsPolicy: [{
+      allowedHeaders: [ '*' ],
+      allowedMethods: [ 'GET' ],
+      allowedOrigins: [ '*' ],
+      exposeHeaders: [ '*' ],
+      maxAgeSeconds: 3000
+    }],
+    policy: JSON.stringify({
```

Comment:
In case there would be more pattern with MediaStore, by default, it's available accessible publicly over HTTPS. [Link](https://docs.aws.amazon.com/mediastore/latest/ug/policies-examples-public-https.html). For the new pattern, it will only allow the specific CloudFront with the `User-Agent` header by default.